homeassistant/components/tessie/update.py

```diff
@@ -29,8 +29,8 @@ async def async_setup_entry(
 class TessieUpdateEntity(TessieEntity, UpdateEntity):
     """Tessie Updates entity."""
 
-    _attr_supported_features = UpdateEntityFeature.PROGRESS
     _attr_name = None
```

Comment:
You also added the translation key for update but you didn't remove this one, correct?

Reply:
Im second guessing the update entity... I need to check how it shows up in the UI. Ill do some tests and then re-open this.

Reply:
Ok, got this straight. Let it have a name again, and fixed the relevant test.